modules/core/include/opencv2/core/mat.inl.hpp

```diff
@@ -1096,6 +1096,12 @@ void Mat::push_back(const Mat_<_Tp>& m)
     push_back((const Mat&)m);
 }
 
+template<> inline
+void Mat::push_back(const MatExpr& expr)
+{
+    push_back((const Mat&)expr);
```

Comment:
Done. Appreciate the feedback.

Do you think this should also be done for "(const Mat&)m" in the other push_back overload (line 1096 of the same file)? The cast there is from a const Mat_<_Tp>& to const Mat&.